Microsoft.Azure.Cosmos/src/Routing/GlobalEndpointManager.cs

```diff
@@ -116,13 +116,15 @@ public Uri GetHubUri()
         /// </summary>
         public static async Task<AccountProperties> GetDatabaseAccountFromAnyLocationsAsync(
             Uri defaultEndpoint,
-            IList<string>? locations,
+            IList<string>? locations,
+            IList<string> accountInitializationCustomEndpoints,
```

Comment:
Should this be:

```suggestion
            IList<string>? accountInitializationCustomEndpoints,
```

or are we creating a default empty list?

Reply:
Yes, the list is created as an empty collection by default. Here is [the code](https://github.com/Azure/azure-cosmos-dotnet-v3/blob/users/kundadebdatta/4236_add_custom_domain_names/Microsoft.Azure.Cosmos/src/ConnectionPolicy.cs#L47) which does the initialization.

Reply:
But could the user assign it to `null`?

Reply:
Yes, the Cx can assign it to `null`. However the place where we set this value has [already has a null check](https://github.com/Azure/azure-cosmos-dotnet-v3/blob/b8788d409c64fa2c9a4d3dd33afc734c7f273fc0/Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs#L840C22-L843) which should cover us.